app/helpers/arclight_helper.rb

```diff
@@ -7,8 +7,8 @@ module ArclightHelper
   # @param [SolrDocument]
   def parents_to_links(document)
     safe_join(Arclight::Parents.from_solr_document(document).as_parents.map do |parent|
-      link_to parent.label, solr_document_path(parent.global_id)
-    end, t('arclight.breadcrumb_separator'))
+      link_to(parent.label, solr_document_path(parent.global_id)) + '</div>'
```

Comment:
I think there still might be some problems with this. At the collection level this seems ok, but navigating further has some problems.

![Screen Shot 2019-08-20 at 3 46 53 PM](https://user-images.githubusercontent.com/1656824/63387258-38e12000-c362-11e9-8875-ced59283dcfd.png)
![Screen Shot 2019-08-20 at 3 45 45 PM](https://user-images.githubusercontent.com/1656824/63387259-3979b680-c362-11e9-9f52-66d63c534229.png)

Reply:
Fixed.